### ISSUE TITLE
Accessibility: The navigation does not have a header or other landmark

### ISSUE THREAD

**Issue by ghost** [opentofu/registry-ui#149]
Currently, the navigation does not have a landmark or header that screen reader users can easily jump to. This should be fixed and all important parts of the page should have a landmark to jump to.
